pkg/assetsclient/github/client.go

```diff
@@ -84,7 +84,7 @@ func (c *realGhClient) ListRelease(ctx context.Context) ([]string, error) {
 	releases := []string{}
 
 	for _, release := range repoRelease {
-		releases = append(releases, *release.Name)
+		releases = append(releases, *release.TagName)
```

Comment:
@guettli, @janiskemper - let's discuss my #254 comments a little bit here.

Reply:
I don't have an opinion here. Except that all involved parties should agree to use `Name` XOR `TagName`. I would like to avoid reading the data from two places.

Reply:
Yes, you are right, let's go with release names for now.